framework/components/ecl-buttons/_buttons.scss

```diff
@@ -1,354 +1,161 @@
-/*
-Buttons
```

Comment:
[W3C](https://www.w3.org/TR/CSS21/ui.html#cursor-props): "pointer" The cursor is a pointer that indicates a link.